R/caveats.R

```diff
@@ -0,0 +1,13 @@
+caveats_box <- function() {
+  box(
+    status = "warning", width = NULL, solidHeader = TRUE,
+    strong("Caveats"),
+    br(),
+    "\U2022 Data only includes graduates who were in sustained employment in the associated tax year, and a graduate's industry is recorded as the industry in which they earnt the most in the associated tax year.",
+    br(),
+    "\U2022 Counts have been rounded to the nearest 5, however percentages are based on raw values.",
```

Comment:
% aren't based on raw values anymore so could this just be changed to "Counts have been rounded to the nearest 5 and earnings are rounded to the nearest £100."

Reply:
No problem - have just updated this in the latest commit